source/type/route.h

```diff
@@ -33,6 +33,7 @@ www.navitia.io
 #include "type/fwd_type.h"
 #include "type/geographical_coord.h"
 #include "type/odt_properties.h"
+#include "type/stop_point.h"
```

Comment:
it's a pointer, a forward declare is enough

Reply:
A forward class doesn't seem to be enough in "indexes.h" : 
![image](https://user-images.githubusercontent.com/523524/63761185-fe770600-c8c0-11e9-93e3-0ce21e822574.png)

Reply:
forward decl is enough in the .h, you'll need to update the `.cpp` with the proper include.

Reply:
Ok done